doc/modules/pipeline.rst

```diff
@@ -148,7 +149,7 @@ object::
     >>> pipe # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
     Pipeline(...,
              steps=[('reduce_dim', PCA(copy=True,...)),
-                    ('clf', SVC(C=1.0,...))])
+                    ('clf', SVC(C=1.0,...))], verbose=False)
```

Comment:
If we set an analogy, `transformer_weights` are shown here in same documentation. They're not having any value in this particular example.
```python
    >>> combined # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
    FeatureUnion(n_jobs=1,
                 transformer_list=[('linear_pca', PCA(copy=True,...)),
                                   ('kernel_pca', KernelPCA(alpha=1.0,...))],
                 transformer_weights=None, verbose=False)
```

We can use this as a tie breaker :laughing: